winui/Rating/Clear-rating.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
```

Comment:
Layout: Post

Reply:
We have follow the small letter for all other controls.